AdventOfCode/UserClasses/DefaultClass.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AdventOfCode.UserClasses
+{
+    class DefaultClass
+    {
+    }
+}
```

Comment:
Sadly I disagree on having a folder and namespace for stuff like this. When I used the template myself last year I simply put the IntcodeComputer etc within the namespace for the appropriate year, and I felt that worked out quite nicely. There's also an argument about keeping the upper levels as bare-bones and free of interaction as possible as I believe that makes it easier for users who are new to a language.

Reply:
I was personally split on it myself. I think it makes good sense from a compartmentalization standpoint, and keeping a fairly clean file structure. Maybe moving it to `AdventOfCode/Solutions/UserClasses` would make sense.  It also allows for year of year items like a generic Graph-like data structure.

Reply:
This is also the change I'm least married to. I think it's a good idea and will use it in my personal copy, but if you think it doesn't work for the template, that's totally fine.

Reply:
For now I don't see the need for it and would rather be without dummy files like this, but I might add some examples to the readme later on.

Reply:
That's fine. I can rip that back out. and re-work the documentation.

Reply:
7054c4c and d0fca95